test/infamy/restconf.py

```diff
@@ -207,7 +207,13 @@ def get_datastore(self, datastore="operational", path="", parse=True):
             dspath = f"{dspath}/{path}"
 
         url = f"{self.restconf_url}{dspath}"
-        return self._get_raw(url, parse)
```

Comment:
Not a fan of this. This will hide errors. You only get 404 when you do something wrong